java/openmldb-taskmanager/src/test/resources/taskmanager.properties

```diff
@@ -14,4 +18,6 @@ batchjob.jar.path=
 # Spark Config
 spark.master=local
 spark.yarn.jars=
-spark.home=
\ No newline at end of file
+spark.home=
+
+max.job.id=3
```

Comment:
This config is for pre-allocated job id, not max job id. Maybe rename to `prefetch.job.num`.

Reply:
OK